### PR TITLE
Tools qa

### PR DESCRIPTION
**Pull request by Elici965**

### CODE REVIEW

File: cypress/integration/locators.spec.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  
+  describe('Fill Form', () => {
+    const personalInformation: PersonalInformation = new PersonalInformation();
+    describe('given a user enters the qavbox page', () => {
+      before(() => {
+        cy.visit('https://qavbox.github.io/demo/signup/');
+      })
+    
+    describe('when the user fills the form', () => {
+      it('when the user fills the form',() => {
```
Comment (salorrego) [inline cypress/integration/locators.spec.ts]: what should be the then here?

Comment (salorrego) [reply cypress/integration/locators.spec.ts]: what are you validating?

**Review by salorrego — CHANGES_REQUESTED** [review]

File: cypress/integration/locators.spec.ts (hunk 1 above)
Comment (Elici965) [reply cypress/integration/locators.spec.ts]: Changes were uploaded.

File: cypress/integration/locators.spec.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    }) 
+
+    describe('then the information is submited.', () => 
+    {
+      it('then the information is submited.',() => 
+      {
+        personalInformation.test();
+      }) 
+    })   
+  })
```
Comment (salorrego) [inline cypress/integration/locators.spec.ts]:
```suggestion
    it('then the information should be submited.',() =>
    {
      personalInformation.test();
    })
  })
```
anything else that tells us that the test is successful?

File: cypress/page/personal-information.page.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    cy.get(this.txtEmail).type("elizabeth.Correa@gmail.com").should('have.value', 'elizabeth.Correa@gmail.com')   
+    cy.get(this.txtTelefono).type("2345432").should('have.value', '2345432')
+    cy.get(this.lstGender).select('Female').should('have.value', 'female')
+    cy.get(this.radioExperience).check('two').should('be.checked').and('have.value','two')
+    cy.get(this.CheckSkills).check('automationtesting').should('be.checked').and('have.value','automationtesting')
+    cy.get(this.btnSubmit).click()
+    cy.on('window:alert',(t)=>{expect(t).to.contains('Registration Done!')})
+  }
+
+  public test() {
```
Comment (salorrego) [inline cypress/page/personal-information.page.ts]: maybe a better name for this, something like `checkIfFormIsSubmited` or `expectFormSubmited`

**Review by salorrego — CHANGES_REQUESTED** [review]

**Pull request closed without merging**